nix/docker-files/docker.nix

```diff
@@ -64,8 +64,8 @@
       finalImageName = "nixos/nix";
       finalImageTag = "2.15.0";
     };
-    genFlake = inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927;
```

Comment:
Aliases for your cabal *executables* only (this excludes benchmarks and test suites) are added to your `packages` and `apps` flake outputs.
Alias means just the executable name as found in your cabal file, without the `$cabal_package_name-exe-` prefix.

For projects supporting a single compiler (like in this repo, `ghc927`), the compiler name is no longer appended to the flake outputs names.

```
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927
->
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927

inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927
->
inputs'.self.packages.dapps-certification-helpers-exe-generate-flake-ghc927

inputs'.self.apps.dapps-certification-exe-plutus-certificatio-ghc927
->
inputs'.self.apps.plutus-certification
```

If you cabal project has multiple executables with the same name across your cabal files, a warning will be generated when running `list-flake-outputs` and no aliases will be created.